src/app/api/v1/share-link/[id]/route.ts

```diff
@@ -9,9 +9,11 @@ import { NextResponse } from "next/server";
 const repo = new SHLinkPrismaRepository(prisma);
 
 export async function GET(request: Request, { params }: { params: { id: string } }) {
+    const url = new URL(request.url);
+    const managementToken = url.searchParams.get('management_token');
```

Comment:
please remove this from this PR, only focus on create shlink endpoint for this PR

Reply:
I had to add that because it was giving errors during testing due to it being a dependency for the add-endpoint